{"name": "Проверка ошибки при создании двух одинаковых курьеров", "status": "passed", "description": "Отправляем повторный запрос на создание курьера, проверяем ответ и удаляем курьера", "start": 1728126240529, "stop": 1728126240622, "uuid": "c2eeb493-bc20-4259-9961-e36fe4bbbe78", "historyId": "d64c0d867f738829cd30884375940b6d", "testCaseId": "d64c0d867f738829cd30884375940b6d", "fullName": "test_create_courier.TestCreateCourier#test_registration_double_courier_failed", "labels": [{"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "Home-PC"}, {"name": "thread", "value": "15328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_courier"}]}